zegami_sdk/annotation.py

```diff
@@ -7,16 +7,18 @@
 import base64
 import io
 import os
-
 import numpy as np
 from PIL import Image
 
+from zegami_ai.mrcnn.utils import get_bool_mask_bounds
```

Comment:
I may remove this and copy the function over to break the dependency on zegami-ai

Reply:
Done in latest commit